app/web/features/auth/logins/LoginCard.tsx

```diff
@@ -79,28 +79,38 @@ export default function LoginsPage({
         <IconText
           icon={LocationIcon}
           text={
-            <Trans t={t} i18nKey="auth:active_logins.location">
-              Near{" "}
-              <strong>
-                {{ approximate_location: session.approximateLocation }}
-              </strong>
+            <Trans
+              t={t}
+              i18nKey="auth:active_logins.location"
+              values={{ approximate_location: session.approximateLocation }}
+            >
+              {`Near `}
```

Comment:
Is there a reason for these bing like this instead of just `Near <strong>...`?

Reply:
Yes typescript didn't like it that way! I upgraded the `@types/react` package as part of this upgrade. It also stopped working the old way. It wanted the translation variable on its own line and then to define the value of the variables to be translated now.